src/server/scripts/Northrend/Ulduar/Ulduar/instance_ulduar.cpp

```diff
@@ -11,6 +11,12 @@
 #include "WorldPacket.h"
 #include "WorldSession.h"
 
+enum class raidDifficulty
+{
+    DIFFICULTY_10MAN = 0,
+    DIFFICULTY_25MAN = 1,
+};
```

Comment:
Is this necessary?

You could just check which diff it is with Is25ManRaid?

Reply:
Well, I don't have to it's true, but then this will have magic code switching between raid difficulty

Reply:
I'm not very good at scripting, so just wanted to understand how it works ^^

Reply:
It's true that having a 0 and 1 for 2 different difficulties is silly since you can type !0, but I think it's something we shouldn't omit since it helps code be read in a simple way

Reply:
Well it would be !Is25ManRaid no?

Like I said, I don't really know. Just trying to learn too :P